website_anchor_smooth_scroll/static/src/js/website_anchor_smooth_scroll.js

```diff
@@ -0,0 +1,27 @@
+/**
+*    Copyright 2016 Antiun Ingeniería S.L. - Jairo Llopis
+*    Copyright 2016 LasLabs Inc.
+*    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+**/
+
+odoo.define('website_anchor_smooth_scroll.enable', function(require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+
+    var smooth_scroll = function(event) {
+        event.preventDefault();
+        var anchor_fragment = event.target.hash;
+
+        // Do this before scrolling so that browser history accurately reflects scroll position at time of click 
+        history.pushState(null, document.title, anchor_fragment);
+
+        $('html, body').stop().animate({
```

Comment:
Yup, which means this also needs to be exported https://github.com/OCA/maintainer-tools/pull/208/files#diff-6d42ed87f53a9be31c0530c5d6ae1e11R18

Best bet would probably be to define this module as `website_anchor_smooth_scroll`, then `return {smoothScroll: smooth_scroll}`